### DISCUSSION TITLE
What's the state of Pixi Animate plugin?

### DISCUSSION THREAD

**Discussion by joao-cesar** [pixijs/pixijs#7913] [General]
Hi!

A few days ago I found out that the plugin repository was updated about a month ago.

So I would like to know if the plugin is still in active development and ready for production.

If so, I will definitely invest my time learning how to use it and I will also spread the word in the Animate community.

I've wanted to use Pixi in Animate for some time.

Regards,
JC

**Comment by bigtimebuddy** [comment]
There are a handful of folks that maintain the Pixi Animate project. It's "active" in the sense that people are using it and we made a release recently. Through the generous contribution from PBS KIDS, we invested a lot to upgrade it to use PixiJS v6 and support tweening.

Please use it and report errors and issues or suggestions to the project: github.com/pixijs/animate

**Reply by joao-cesar** [reply]
That's great to hear, Matt!

So I'll deep dive into it.

I took a quick look at the files from the repo but I didn't see any sample of a FLA game or app.

Is there a reference somewhere?

I was able to run some code of my own but I'm still lost in some areas.

**Reply by andrewstart** [reply]
We don't have any examples handy, just the runtime docs here: https://pixijs.io/animate/

**Reply by joao-cesar** [reply]
Thanks, Andrew!
